packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -67,6 +73,14 @@ export class Snackbar implements ComponentInterface {
       this.handleSnackbarHide(e)
     );
     this.configureTimeout();
+    if (this.inoStayVisibleOnHover) {
+      this.snackbarElement.addEventListener('mouseenter', () => {
+        if (this.timeout) clearTimeout(this.timeout);
+      });
+      this.snackbarElement.addEventListener('mouseleave', () => {
+        this.configureTimeout();
+      });
```

Comment:
Can be shortened as well:
```suggestion
      this.snackbarElement.addEventListener('mouseleave', this.configureTimeout);
```

Reply:
Applied